--- Makefile	2025-04-20 20:39:11
+++ Makefilepatched	2025-05-01 22:13:16
@@ -1,6 +1,6 @@
 CXX = g++
 # Stderr logging can be enabled with submission so i will leave the -DDEBUG_PRINT
-CXXFLAGS = -std=c++20 -Wall -Wextra -pedantic -I src -g -DDEBUG_PRINT
+CXXFLAGS = -std=c++20 -Wall -Wextra -pedantic -I src -g -DDEBUG_PRINT -include cstdint -include algorithm
 SRC_DIR := src
 OBJ_DIR := obj
 TEST_DIR := tests
